src/main/sensors/battery.h

```diff
@@ -31,11 +31,11 @@
 #define CELL_VOLTAGE_FULL_CV 420
 
 #define VBAT_CELL_VOTAGE_RANGE_MIN 100
-#define VBAT_CELL_VOTAGE_RANGE_MAX 500
+#define VBAT_CELL_VOTAGE_RANGE_MAX 504
```

Comment:
12 x 42 
MAX_AUTO_DETECT_CELL_COUNT * CELL_VOLTAGE_FULL_CV

Reply:
VBAT_CELL_VOTAGE_RANGE_MAX is simply maximum configurable cell voltage. It is not related to cell count.

Cell count autodetection is problematic for large cell count. Count is determined as `(<pack voltage> / maxcellvoltage) +1`, so cell detection will fail when pack is discharged by more than maxcellvoltage.

MAX_AUTO_DETECT_CELL_COUNT sets limit to autodetection, but increasing it is bad idea

Reply:
this was the only reason i modded _MAX
`rc/main/cli/settings.c:892`:`    { "vbat_full_cell_voltage",     VAR_UINT16  | MASTER_VALUE, .config.minmaxUnsigned = { VBAT_CELL_VOTAGE_RANGE_MIN, VBAT_CELL_VOTAGE_RANGE_MAX }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, vbatfullcellvoltage) },`

Reply:
That would mean that single cell can go to 5.04V.

Reply:
i see, i mistook "full" for pack. TY, understood now.